core/global-options/README.md

```diff
@@ -6,7 +6,7 @@
 
 ### `--concurrency`
 
-How many threads to use when Lerna parallelizes the tasks (defaults to `4`)
+How many threads to use when Lerna parallelizes the tasks (defaults to count of logical CPU cores)
```

Comment:
You'll need to update the option configuration's [`defaultDescription`](https://github.com/lerna/lerna/blob/5c1b84c699d80dbd8c5cce009aae8b82569e32a8/core/global-options/index.js#L14) to reflect this, as well.

Reply:
Sure, I will update the `defaultDescription`